docs/changes/units/16087.feature.rst

```diff
@@ -0,0 +1,10 @@
+Add a ``format_spec`` argument to the ``to_string`` method of the ``Quantity`` class. Enables custom number formatting, especially useful for consistent notation.
+The argument accepts standard Python format specifiers, allowing detailed control over the output format.
+
+Example usage:
```

Comment:
I think this part can be omitted - that's for the what's-new. You may also want to check the regular documentation - I don't quite recall if we discuss `.to_string()`, but if we do, it might be nice to add an example there as well.